api/src/pcapi/routes/backoffice_v3/offer_validation_rules/blueprint.py

```diff
@@ -137,3 +141,152 @@ def create_rule() -> utils.BackofficeResponse:
         flash(f"Erreur dans la création de la règle : {err}", "warning")
 
     return redirect(url_for("backoffice_v3_web.offer_validation_rules.list_rules"), code=303)
+
+
+@offer_validation_rules_blueprint.route("/<int:rule_id>/delete", methods=["GET"])
+def get_delete_offer_validation_rule_form(rule_id: int) -> utils.BackofficeResponse:
+    rule_to_delete = offers_models.OfferValidationRule.query.get(rule_id)
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=empty_forms.EmptyForm(),
+        dst=url_for("backoffice_v3_web.offer_validation_rules.delete_rule", rule_id=rule_id),
+        div_id=f"delete-offer-validation-rule-{rule_id}",  # must be consistent with parameter passed to build_lazy_modal
+        title="Supprimer une règle de validation d'offre",
+        button_text="Supprimer la règle",
+        information=f"La règle {rule_to_delete.name} et ses sous-règles seront définitivement supprimées de la base de données. Veuillez confirmer ce choix.",
+    )
+
+
+@offer_validation_rules_blueprint.route("/<int:rule_id>/delete", methods=["POST"])
+def delete_rule(rule_id: int) -> utils.BackofficeResponse:
+    rule_to_delete = offers_models.OfferValidationRule.query.get(rule_id)
+    subrules_to_delete = offers_models.OfferValidationSubRule.query.filter_by(validationRuleId=rule_id).all()
+
+    if rule_to_delete:
+        try:
+            # ondelete CASCADE doesn't work, so this
+            for pouet in subrules_to_delete:
```

Comment:
C'est quoi `pouet` ? pouet pouet

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏```python
for subrule in subrules_to_delete: 
```

Reply:
C'est pour ça que c'est important la relecture 😁